api/v1alpha2/cluster_types.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2019 The Kubernetes Authors.
+Copyright The Kubernetes Authors.
```

Comment:
This is going to need the year in the copyright even though it's a generated file it's not meant to be re-generated. This is true for all the generated files not destined to be regularly-regenerated below as well. And for the actual generated files (deep copy) you will need to either: 1) extend the regex in the python validation script or 2) add the words `DO NOT EDIT` to the file

Reply:
Fixed the copyright in these files and the controllers/* ones. I opened https://github.com/kubernetes-sigs/controller-tools/pull/306 to address the deepcopy issue

Reply:
much appreciated.